jib-core/src/main/java/com/google/cloud/tools/jib/frontend/HelpfulSuggestions.java

```diff
@@ -108,6 +108,24 @@ public String forDockerNotInstalled() {
     return suggest("make sure Docker is installed and you have correct privileges to run it");
   }
 
+  /**
+   * @param parameter the parameter name (should be 'to.image' for gradle, {@literal <to><image>}
```

Comment:
We prob shouldn't include specificities for the caller here, since if we change the parameter name, this comment would need to be changed. Having an example is good though.